src/style.css

```diff
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+body {
+  padding: 0;
+}
+
+.form {
+  margin-top: 20px;
+  display: flex;
+}
+
+.bar--big {
+  height: 70px;
+}
+
+.bar {
+  display: flex;
+  width: 100%;
+  padding-left: 62px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-x: 26px;
+  background-position-y: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
```

Comment:
2'nd point in checklist, you should create one base class, bar in your example, and have only one more class, as you have bar--small, where you will override needed styles